src/resources/views/video/_create_modal.blade.php

```diff
@@ -0,0 +1,30 @@
+<div id="create-video-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-md" role="document">
+        <div class="modal-content">
+            {!! Form::open([
+                    'url'  => route('vanilo.admin.video.store'),
+                    'method' => 'POST'
+                ])
+            !!}
+
+            <div class="modal-header">
+                <h5 class="modal-title">{{ __('Add Video') }}</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="{{ __('Close') }}"></button>
+            </div>
+
+            <div class="modal-body">
+                {{ Form::hidden('for', shorten(get_class($model))) }}
```

Comment:
[Since PHP 8.0 - 2020](https://kinsta.com/blog/php-8/#class-syntax-on-objects) we can simply do this:

```suggestion
                {{ Form::hidden('for', shorten($model::class)) }}
```